crd-docs/crd/kiali.io_kialis.yaml

```diff
@@ -1022,6 +1022,18 @@ spec:
                             secret_name:
                               description: "The name of the secret that contains the credentials necessary to connect to the remote cluster. This secret must exist in the Kiali deployment namespace."
                               type: string
+                      kiali_urls:
```

Comment:
same in "roles/default/kiali-deploy/defaults/main.yml" and in helm repo?

Reply:
You could. I personally like to have all the values in there because then it explicitly gets set in the ConfigMap. So if you are asking me, I would say, yes, add it in there, too.